lib/CredentialIssuancePurpose.js

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+const { AssertionProofPurpose } = require('jsonld-signatures').purposes;
+
+class CredentialIssuancePurpose extends AssertionProofPurpose {
+}
```

Comment:
We need to check that `issuer`, if present in the VC, matches the `controller` of the verification method.

Reply:
I would prefer this to be fixed prior to pulling this in, but a quick follow-on PR would also be acceptable.

Reply:
> We need to check that issuer, if present in the VC, matches the controller of the verification method.

Is that always going to be the case?

Reply:
Yeah, if someone says `issuer: x` then there needs to be an `assertionMethod` proof with a `verificationMethod` controlled by `x` or the VC is not valid.